beater/jaeger/http.go

```diff
@@ -42,6 +43,7 @@ const (
 
 var (
 	httpRegistry      = monitoring.Default.NewRegistry("apm-server.jaeger.http")
+	monitoringKeys    = append(interceptors.MetricsMonitoringKeys, request.IDEventDroppedCount)
```

Comment:
```suggestion
	monitoringKeys    = append(interceptors.MetricsMonitoringKeys, request.IDEventReceivedCount)
```

AFAICS we're not using IDEventDroppedCount (maybe we were in the past), but we are using IDEventReceivedCount.